src/index-hindi.html

```diff
@@ -0,0 +1,655 @@
+        <button class="btn btn-info btn-lg sendResponse" id="bottomSend">Send your response</button>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <hr>
+      <div class="row">
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">What is Net Neutrality?</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/what-is-net-neutrality/">Read more...</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">Facebook's Free Basics</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/free-basics/">Our coverage of Free Basics</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">#SaveTheInternet Blog</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/">Updates from the SaveTheInternet team</a>
+            </div>
+          </div>
+        </div>
+      </div>
+  </div>
+  <footer>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12 text-center">
+          <p>
+            <a href="https://github.com/netneutrality/" role="button" class="btn btn-sm btn-info">Made with ♥ by folks just like you</a>
+        </div>
+      </div>
+    </div>
+  </footer>
+
+    <style>
+      <!-- got bored of scrolling up all the way -->
+
+      #response-copied { display: none; }
+
+    </style>
+
+      <!-- Modal -->
+      <div class="modal fade" id="sendModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title"> प्रतिक्रिया भेजें।</h4>
+            </div>
+            <div class="modal-body">
+              <label for="responseSubject">
+                <h4>विषय</h4>
+                <input type="text" id="responseSubject" readonly class="form-control"></label>
+              <h4>आप का जवाब</h4>
+              <textarea class="form-control" id="responseTextarea" readonly></textarea>
+              <h4 id="copy-instruction">ऊपर लिखा हुआ टेक्स्ट कॉपी करें<span class="copy-short"> by Right-click->Copy (or <span class="control-str">Ctrl</span>+C)</span> <button id="copyBtn" class="btn btn-primary">हो गया</button></h4>
```

Comment:
Lekh is also = Paragraphs. A lot of hindi speaking users might not understand `text` na.